.github/workflows/gluten.yml

```diff
@@ -56,20 +56,29 @@ jobs:
         id: gluten-cache
         uses: actions/cache@v3
         with:
-          path: gluten/package/target/
-          key: gluten_package_${{ steps.date.outputs.date }}
+          path: gluten/dist/
+          key: gluten_dist_${{ steps.date.outputs.date }}
       - name: Build gluten project
         run: |
           if [[ "${{ steps.gluten-cache.outputs.cache-hit }}" != 'true' ]]; then
             git clone https://github.com/oap-project/gluten.git
```

Comment:
```suggestion
            git clone https://github.com/apache/incubator-gluten.git
```

Reply:
wait for https://github.com/apache/incubator-gluten/pull/5193